src/translations/fr.json

```diff
@@ -161,6 +161,7 @@
     "FreeProps": "Propriétés de l'ouvrage",
     "SubstationFreeProps": "Propriétés du site de l'ouvrage",
     "AddFreePropCrit": "Ajouter un filtrage par propriété",
+    "AddDescription": "Ajouter une description (optionnel)",
```

Comment:
No need to create a new label when "descriptionProperty" was used for the same thing

Reply:
Description Property:  is the label used to describe the purpose or content of an item.

AddDescription  This label will be attached to a clickable link that users can interact with to add a description.